search-app/src/cmr/search/site/data.clj

```diff
@@ -66,18 +69,33 @@
         (:items data)
         (sort-by #(get-in % [:umm :EntryTitle]) data)))
 
-(defmethod collection-data :default
+(defn-timed get-collection-data
+  "Get the collection data from elastic by provider id and tag. Sort results
+  by entry title"
   [context tag provider-id]
-  (as-> {:tag-key tag
-         :provider provider-id
-         :result-format {:format :umm-json-results}} data
-        (query-svc/make-concepts-query context :collection data)
-        (assoc data :page-size :unlimited)
-        (query-exec/execute-query context data)
-        (:items data)
-        (sort-by #(get-in % [:umm :EntryTitle]) data)))
+  (let [conditions (query-svc/generate-query-conditions-for-parameters
+                    context
+                    :collection
+                    {:tag-key tag
+                     :provider provider-id})
+        query (query-model/query {:concept-type :collection
+                                  :condition (gc/and-conds conditions)
+                                  :skip-acls? false
+                                  :page-size :unlimited
+                                  :result-format :query-specified
+                                  :result-fields [:concept-id
+                                                  :doi-stored
+                                                  :entry-title
+                                                  :short-name
+                                                  :version-id]})
+        result (query-exec/execute-query context query)]
+    (sort-by :entry-title (:items result))))
+
+(defn-timed collection-data :default
```

Comment:
Isn't this a syntax error? why change `defmethod` to `defn-timed`?

Reply:
Yes I had it in a separate method and must have done an undo accidentally or something right before committing. I'm going to fix now.

Reply:
Looks like I hadn't saved what I was doing. Fixed. Thanks!!